src/pydata_sphinx_theme/assets/styles/variables/_layout.scss

```diff
@@ -3,4 +3,11 @@
   * Overall Layout Variables
   */
   --pst-header-height: 60px;
+
+  /*****************************************************************************
+  * control over object depth
+  */
+  --pst-level-1: 1000;
```

Comment:
How about we just re-use the levels from Bootstrap so that we aren't re-defining our own ones? https://getbootstrap.com/docs/5.0/layout/z-index/